node/consensus/data/message_validators.go

```diff
@@ -59,7 +59,16 @@ func (e *DataClockConsensusEngine) validateTxMessage(peerID peer.ID, message *pb
 			e.logger.Debug("could not unmarshal token request", zap.Error(err))
 			return p2p.ValidationResultReject
 		}
-		// NOTE: There are no timestamps to be validated for token requests.
+		if tx.Timestamp == 0 {
+			// NOTE: The timestamp was added in later versions of the protocol,
```

Comment:
We also _could_ turn this into a breaking change for the protocol in order to cull all of the lingering messages which are not timed out.

Or we make it such that this little escape hatch expires after a particular calendar date (12 hours from release ?) and then messages without a timestamp are automatically ignored.